src/components/Log/AddCatalog.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <ion-page>
+    <ion-content>
+      <ion-header>
+        <ion-toolbar>
+          <ion-buttons slot="start">
+            <ion-back-button default-href="/tabs/log"></ion-back-button>
+          </ion-buttons>
+          <ion-title>
+            Add Catalog
+          </ion-title>
+        </ion-toolbar>
+      </ion-header>
+      here are the items to add
+    </ion-content>
+  </ion-page>
+</template>
+
+<script>
+import {
+  IonPage,
+  IonBackButton,
+  IonButtons,
+  IonToolbar,
+  IonContent,
+  IonTitle,
+  IonHeader
+} from '@ionic/vue';
+
+export default {
+  components: {
+    IonPage,
+    IonBackButton,
+    IonButtons,
+    IonContent,
+    IonToolbar,
+    IonTitle,
+    IonHeader
```

Comment:
Why do they all have `Ion` in front of their names? If they all have something in common like that, can't you just make a overall name / structure of some sort and put them all inside?

Reply:
Ionic recommends to incrementally include all custom Ionic components on a need basis inside each Single File Component. Consolidating may be a consideration given the fact that it is verbose, but i think the key is splitting the larger components that I built off into smaller sub components instead of creating some sort of namespace that includes the components automatically

Reply:
Okie dokie. Sounds good to me!